notebooks/DrizzlePac/align_mosaics/align_mosaics.ipynb

```diff
@@ -0,0 +1,940 @@
+    "x_cord, y_cord = match_tab_chip2['col11'], match_tab_chip2['col12']\n",
+    "plt.scatter(x_cord, y_cord, s = 50, edgecolor = 'r', facecolor = 'None',label='Matched Sources, Chip 2')\n",
+    "plt.ylim(0,2051)\n",
+    "plt.xlim(0,4096)\n",
+    "plt.legend(loc = 'best', fontsize = 20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# v06a vector residuals\n",
+    "Image(filename='vector_ick906l5q_flc.png', width =600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# v06a fit residuals\n",
+    "Image(filename='residuals_ick906l5q_flc.png', width =600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6c. Rerun TweakReg to update the UVIS FLC headers\n",
+    "\n",
+    "Now that the parameters have been fine-tuned and the fits look adequate, TweakReg is run a second time to update the image header WCS keywords by setting the parameter `updatehdr` to True. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tweakreg.TweakReg('*flc.fits',\n",
+    "                  imagefindcfg={'threshold':200,'conv_width':3.5,'dqbits':~4096},\n",
+    "                  refimage='f160w_drz_sci.fits', \n",
+    "                  refimagefindcfg={'threshold':50,'conv_width':2.5}, \n",
+    "                  searchrad=5.0, ylimit=0.6,\n",
+    "                  shiftfile=False, updatehdr=True, wcsname='UVIS_FLC', interactive=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 7. Drizzling the F657N mosaic\n",
+    "\n",
+    "The full set of 12 FLC frames have now been aligned and may be drizzled to a mosaic exactly half the scale (0.04\"/pixel) of the original IR mosaic (0.08\"/pixel). The same output WCS paramters: `final_rot`,`final_ra`, and `final_dec` values are used, but the `final_outnx` and `final_outny` are now twice the size of the IR mosaic at 8000x8400 pixels. \n",
+    "\n",
+    "To save time for this notebook, the visit-level cosmic ray flags are assumed to be adequate, Steps 3, 4, 5, 6 are turned off, and the parameter `resetbits` is set to '0' to avoid wiping out the 4096 flags in the FLC data quality arrays. \n",
+    "\n",
+    "To further improve cosmic-ray rejection in the chip gap, AstroDrizzle may alternatively be run with all steps turned on, as shown below.\n",
+    "\n",
+    ">astrodrizzle.AstroDrizzle('*flc.fits', output='f657n', preserve=False, clean=False, build=False, context=False, resetbits=0, skymethod='match', driz_sep_bits='64,32', combine_type='minmed', final_bits='64,32', final_wcs=True, final_scale=0.04, final_pixfrac=1.0, final_rot=-35, final_ra=274.721587, final_dec=-13.841549, final_outnx=8000, final_outny=8400)"
```

Comment:
Why is this segment necessary?  It looks like it's the same as what's in the cell below.  Maybe it's just a typo and can be deleted?

Reply:
This was a little confusing, but there was a difference. I've removed the full astrodrizzle call in the markdown cell and replaced it with the four parameters that should be changed in the code cell below it instead.